core/src/test/java/org/apache/accumulo/core/util/TimerTest.java

```diff
@@ -89,21 +85,26 @@ public void testElapsedPrecision() throws InterruptedException {
     Thread.sleep(sleepMillis);
 
     long elapsedMillis = timer.elapsed(MILLISECONDS);
-    assertEquals(sleepMillis, elapsedMillis, 5, "Elapsed time in milliseconds is not accurate.");
+    assertTrue(elapsedMillis >= sleepMillis, "Elapsed time in milliseconds is not correct.");
```

Comment:
I think with this change to the test case, we could just remove the whole thing since now this functionality is covered in other test cases.

Reply:
Test removed in aa79abd